CHANGELOG.md

```diff
@@ -26,6 +26,27 @@ Classify the change according to the following categories:
     ### Removed
 
 
+## gridRE-dev
```

Comment:
is the plan to merge #456 into this and then merge together or to merge one first?

Reply:
I was thinking we would merge this one first if possible, because we might not get to that one until mid January. That PR would address an issue that already exists with all of the % RE calculations, whereas this one adds new RE outputs (that do also have the whole total load issue). Do you think that's an okay approach?

Reply:
Yes but another thought is that if there is a known issue that we aren't fixing right away I wonder if it makes sense to temporarily remove the affected total load related outputs and/or note it somewhere for users. Mid Jan is pretty soon though especially with the holidays so maybe not worth the time to do that.